python/lsst/ts/idl/enums/MTHexapod.py

```diff
@@ -11,20 +11,64 @@
 # the Free Software Foundation, either version 3 of the License, or
 # (at your option) any later version.
 #
-# This program is distributed in the hope that it will be useful,
+# This program is distributed in the hope that it will be useful
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
 
-__all__ = ["ControllerState", "OfflineSubstate", "EnabledSubstate", "ApplicationStatus"]
+__all__ = [
+    "ApplicationStatus",
+    "ControllerState",
+    "EnabledSubstate",
+    "OfflineSubstate",
+    "SalIndex",
+]
+
+import enum
+
 
 # MTHexapod and MTRotator have the same enum values for
-# ControllerState, OfflineSubstate, EnabledSubstate and ApplicationStatus
+# ControllerState, OfflineSubstate, and EnabledSubstate.
 from .MTRotator import (
     ControllerState,
-    OfflineSubstate,
     EnabledSubstate,
-    ApplicationStatus,
+    OfflineSubstate,
 )
+
+
+class ApplicationStatus(enum.IntFlag):
+    """Bit masks for the value reported in ``telemetry.application_status``.
+
+    These are from https://github.com/lsst-ts/ts_hexapod_controller/blob/master/include/actuatorTlm.h  # noqa
+    """
+
+    FOLLOWING_ERROR = 0x1
+    MOVE_COMPLETE = 0x2
+    HEX_MOVE_COMPLETE_MASK = 0x2  # Backwards compatible name
```

Comment:
Why have the backwards compatibility here but not in the MTRotator.py file?

Reply:
Because this value is specific to the hexapod. There is no such flag for MTRotator.

Originally I dug these constants out of C/C++ code and they seemed to be identical for MTRotator and MTHexapod so I had one enum class shared between the two. However, there is at least one value that has different names (just one, I think) and a few that are only in one or the other so it's better to have two independent classes. I should have done that from the start.

Reply:
That explains it all, thanks.